src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -153,7 +161,7 @@ protected async Task<List<WorkloadDownload>> GetDownloads(IEnumerable<WorkloadId
 
         protected IEnumerable<WorkloadId> GetInstalledWorkloads(bool fromPreviousSdk)
         {
-            var currentFeatureBand = new SdkFeatureBand(_sdkVersion);
+            var currentFeatureBand = new SdkFeatureBand(_installedFeatureBand.ToString());
```

Comment:
I wonder if you need the `_currentFeatureBand` here, you can probably just use the `_installedFeatureBand` instead